server_admin/topics.adoc

```diff
@@ -16,10 +16,6 @@ include::topics/authentication/kerberos.adoc[]
 include::topics/authentication/x509.adoc[]
 include::topics/authentication/webauthn.adoc[]
 include::topics/sso-protocols.adoc[]
-include::topics/sso-protocols/oidc.adoc[]
-include::topics/sso-protocols/saml.adoc[]
-include::topics/sso-protocols/saml-vs-oidc.adoc[]
-include::topics/sso-protocols/docker.adoc[]
```

Comment:
Can you move these includes to the sso-protocols.adoc file?  That is the assembly where they go. 
Eventually I think that topics.adoc will only include the chapter head sections, like sso-protocols.adoc